src/components/product/ProductAdminList.js

```diff
@@ -17,8 +17,18 @@ class ProductAdminList extends Component {
 
   componentDidMount() {
     const { navigation } = this.props;
-    const store = navigation.getParam('store', {});
-    this.props.productsFetchByStoreId(store.uid);
+    const storeId = navigation.getParam('storeId', {});
+    this.props.storeFetchById(storeId)
+    this.props.productsFetchByStoreId(storeId);
+  }
+
+  componentDidUpdate(prevProps) { 
+    const { navigation } = this.props;
+    const storeId = navigation.getParam('storeId', {});
+    if(this.props.products.length !== prevProps.products.length){
+      this.props.storeFetchById(storeId)
```

Comment:
it is necessary get the store again when the size of the products changes?

Reply:
No, it is not necessary and I removed it from ComponentdidUpdate because it is only needed once when entering a restaurant